moml/simulation/qm/parser/orca_parser.py

```
@@ -14,10 +14,11 @@
 import concurrent.futures
 import json
 from typing import Dict, List, Optional, Tuple, Union, Any
+from pathlib import Path
```

Ruff (0.11.9)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)